releng/platform-definitions/platform-definition-2022-09/pom.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-   Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
-   Copyright (c) 2022, Datadog, Inc. All rights reserved.
+   Copyright (c) 2022, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Not necessary, no change in file.

Reply:
Ah indeed it was necessary as I added a custom repository in the past but it has been removed along the way.